wasm_utils/src/api_serialization/validation.rs

```diff
@@ -0,0 +1,34 @@
+use holochain_core_types::{chain_header::ChainHeader, hash::HashString};
+
```

Comment:
Maybe there will be another LinkValidationData for links? Since ValidationData doesn't seem to include LinkAction?

Reply:
Yep, there might be another one for links..